### PR TITLE
fix @fluvio/native cross-platform install

### PR DESCRIPTION
**Pull request by Ryanmtate**
This PR fixes the installation of @fluvio/native and @fluvio/client; now correctly installing the platform specific built node module.

### CODE REVIEW

File: native/src/linux/package.json [hunk 1]
```diff
@@ -1,11 +1,14 @@
 {
   "name": "@fluvio/native-linux",
-  "version": "0.1.0-alpha.3",
+  "version": "0.1.0-alpha.5",
```
Comment (sehz) [inline native/src/linux/package.json:3]: how do we update the version next time?

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
     "name": "@fluvio/client",
-    "version": "5.0.0-alpha.1",
+    "version": "5.0.0-alpha.7",
```
Comment (sehz) [inline package.json:3]: seems to be different version from others?

**Review by sehz — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (Ryanmtate) [reply package.json:3]: yeah, had some debugging; had to bump the version to make sure worked.

File: native/src/linux/package.json (hunk 1 above)
Comment (Ryanmtate) [reply native/src/linux/package.json:3]: There are now separate workflows for building and publishing the native platform specific modules; another for the @fluvio/native that likely will not change much, and then the @fluvio/client which will only change if there are changes to the interface.

The platform modules need to be published before the client, if changes are made.

I'll write some documentation for developing tomorrow. For now, the publishing has been run manually, I am going to remove the publish on tags for the @fluvio/native and platform specific modules. I think it is is easier to sequence these manually for now.

Comment (Ryanmtate) [reply native/src/linux/package.json:3]: commit 58d0362 changes so only the @fluvio/client module will be published on tagged release; The other modules will need to be manually dispatched when changes are made.

**Review by sehz — APPROVED** [review]

**Pull request merged**